qa/L0_server_status/server_status_test.py

```diff
@@ -111,17 +118,30 @@ def test_model_latest_infer(self):
 
             # Initially there should be no version stats..
             try:
-                for pair in [("localhost:8000", ProtocolType.HTTP), ("localhost:8001", ProtocolType.GRPC)]:
-                    server_status0, req_id0 = _get_server_status(pair[0], pair[1], model_name)
-                    self.assertTrue(model_name in server_status0.model_status,
-                                    "expected status for model " + model_name)
-                    self.assertEqual(len(server_status0.model_status[model_name].version_status), 3,
-                                    "expected status for 3 versions for model " + model_name)
+                for pair in [("localhost:8000", "http"), ("localhost:8001", "grpc")]:
+                    if pair[1] == "http":
+                        triton_client = httpclient.InferenceServerClient(
+                            url=pair[0], verbose=True)
+                    else:
+                        triton_client = grpcclient.InferenceServerClient(
+                            url=pair[0], verbose=True)
+
+                    self.assertTrue(triton_client.is_server_live())
+                    self.assertTrue(triton_client.is_server_ready())
+                    model_metadata = triton_client.get_model_metadata(model_name)
+                    model_metadata = triton_client.get_model_metadata(
```

Comment:
Put 132-140 in the loop below so that you check the metadata for all 3 versions. You should check that the metadata name and version are both what you expect.